src/plugins/passwd/README.md

```diff
@@ -50,10 +50,13 @@ If the config key `index` is set to `name` passwd entries will be sorted by name
 ```sh
 kdb mount /etc/passwd system:/passwd passwd index=name
 kdb export system:/passwd/root
-#> gecos = root
-#> gid = 0
-#> home = /root
-#> passwd = x
-#> shell = /bin/zsh
-#> uid = 0
+# gecos = root
```

Comment:
can you maybe write a regex that ignores values (which might be different from system to system)

Reply:
Unfortunately, I could not come up with a regex that works and is not too complicated to read.